src/modules/auth/ui/views/sign-in-view.tsx

```
@@ -0,0 +1,172 @@
+import { useState } from "react"
+import { auth } from "@/lib/auth"
+
+const formSchema= z.object({
+    email:z.string().email(),
+    password: z.string().min(1, { message:"Password is Required" })
+})
+
+export const SignInView = () => {
+
+    const router=useRouter();
+    const [error,setError]=useState<String | null>(null);
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)